app/models/plans.py

```
@@ -0,0 +1,74 @@
+"""SQLAlchemy models for weekly and day plans."""
+
+from __future__ import annotations
+
+from datetime import date, datetime
+from typing import Dict
+
+from sqlalchemy import CheckConstraint, Date, DateTime, ForeignKey, Integer, JSON, String
```

Flake8 (7.3.0)

[error] 'sqlalchemy.String' imported but unused

(F401)